testrunner/FFOSBR.js

```diff
@@ -12,67 +12,78 @@
 
 },{"./settings":2}],2:[function(require,module,exports){
 function Settings() {
-  var timeInMilliSec = 0;
 
   var options = {
-    photos: false,
-    videos: false,
-    contacts: false,
-    text: false,
-    intervalTime: 0, // pass in value in hours
+    photos: true,
+    videos: true,
+    contacts: true,
+    text: true,
+    intervalTime: 24, // pass in value in hours
     id: 0,
     registeredTimer: false,
-    repeat: false
+    repeat: true
   };
 
+  // Load options if present
```

Comment:
Any reason this "constructor" code in an ASCF? Typically, this should be in the body of your Settings class, similar to your "timeInMilliSec" and "options".